javascript/rexuws/package.json

```
@@ -0,0 +1,14 @@
+{
+    "scripts": {
+        "run": "node app.js"
+    },
+    "dependencies": {
+        "rexuws": "^1.1.4",
```

```suggestion
        "rexuws": "~1.1.4",
```

the idea is to have the same operation type in all package.json for javascript (not sure which is more standard)

---

Sure, makes sense to limit it to only minor version changes.